yarn.lock

```diff
@@ -5683,20 +5683,6 @@ http-proxy-middleware@0.19.1:
   version "0.19.1"
   resolved "https://registry.yarnpkg.com/http-proxy-middleware/-/http-proxy-middleware-0.19.1.tgz#183c7dc4aa1479150306498c210cdaf96080a43a"
   integrity sha512-yHYTgWMQO8VvwNS22eLLloAkvungsKdKTLO8AJlftYIKNfJr3GK3zK0ZCfzDDGUBttdGc8xFy1mCitvNKQtC3Q==
-  dependencies:
```

Comment:
Same comment as https://github.com/openshift-assisted/assisted-ui-lib/pull/451#discussion_r577476247

Reply:
`yarn.lock` has been removed from the PR